src/app/containers/Ad/Amp/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+const constructAdJsonData = ({ id, service }) => {
+  const data = {
+    targeting: {
+      slot: 'leaderboard',
+      asset_type: 'index',
+      story_id: id,
+      channel: service,
+      sectn: service,
+      subject: '',
```

Comment:
url subsection, example ‘internacional’ for https://www.bbc.com/mundo/amp/noticias-internacional-51518722